app/jinja_filters.py

```diff
@@ -190,19 +190,38 @@ def setAttributes(dictionary, attributes):
 
 
 @blueprint.app_template_filter()
-def question_as_legend(question):
+def answers_require_legend(question):
     question_type = question['type']
 
     if question_type == 'MutuallyExclusive':
-        return True
+        return False
 
     answers = question['answers']
     more_than_one_question = len(answers) > 1
 
-    if more_than_one_question:
+    if more_than_one_question and question_type != 'DateRange':
         return True
 
-    answer = answers[0]
+    return False
+
+
+@blueprint.app_context_processor
+def answers_require_legend_processor():
+    return dict(answers_require_legend=answers_require_legend)
+
+
+@blueprint.app_template_filter()
+def answer_requires_legend(question):
+    if answers_require_legend(question):
+        return False
+
+    question_type = question['type']
+
+    if question_type == 'MutuallyExclusive':
```

Comment:
If it can, it would make sense to make this the first check in the filter, so it can return early without calling answers_require_legend.